config/initializers/devise.rb

```diff
@@ -142,7 +142,7 @@
 
   # ==> Configuration for :validatable
   # Range for password length.
-  config.password_length = 8..128
+  config.password_length = (Utils.if_present(ENV['MIN_PASSWORD_LENGTH'], :to_i) || 8)..128
```

Comment:
If `ENV['MIN_PASSWORD_LENGTH']` is `''`, will this end up as `0`?  Might be a bit dangerous.